sormas-e2e-tests/src/test/resources/features/sanity/web/Contacts.feature

```diff
@@ -131,3 +131,15 @@ Feature: Contacts end to end tests
     And I am accessing the contacts from New Visit
     And I open Follow up Visits tab from contact directory
     Then I am validating the From and To dates displayed
+
+  @issue=SORDEV-5490
+  Scenario: Create a simple contact and convert it into a case
```

Comment:
Create a contact and create a case for contact person - rename it like this, because contacts are not converted :D